config.go

```diff
@@ -38,7 +38,7 @@ var (
 	ClientVersion = "2023-07-06"
 
 	// Agent version to control agent rollover.
-	AgentVersion = "2023-07-12b"
+	AgentVersion = "2023-07-14a"
```

Comment:
nit: by convention, just `2023-07-14` when there's no additional change made on that date.